tests/5.1/default/test_default_firstprivate_task.c

```diff
@@ -0,0 +1,58 @@
+//--------------- test_default_firstprivate_task.c -------------------------//
+//
+// OpenMP API Version 5.1 Aug 2021
+//
+// Tests the behavior of the default clause when the specified data-sharing attribute is firstprivate. This test focuses 
+// on the use of the task construct. Firstprivate should allow any data initialized outside the task region to return to its original value
+// after being manipulated within the task region.
+
```

Comment:
https://github.com/SOLLVE/sollve_vv/blob/master/tests/5.0/declare_target/test_declare_target_nested.c
Try to make the header matchup to this one, with a border at the top and bottom, and the length of text and border not exceeding ~82 characters.